tstest/integration/vms/vms_test.go

```diff
@@ -280,6 +294,8 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 		t.Skip("not running integration tests (need -run-vm-tests)")
 	}
 
+	os.Setenv("CGO_ENABLED", "0")
```

Comment:
XXX: If I don't do this then the binaries that are created will have their dynamic link loader set to somewhere that is not in the test virtual machines.

Reply:
Move this to BuildTestBinaries then?

Reply:
Done